targets/TARGET_STM/sleep.c

```diff
@@ -199,6 +200,7 @@ void hal_deepsleep(void)
 #else /* TARGET_STM32L4 */
     HAL_PWR_EnterSTOPMode(PWR_LOWPOWERREGULATOR_ON, PWR_STOPENTRY_WFI);
 #endif /* TARGET_STM32L4 */
+    mbed_sdk_inited = 0;
```

Comment:
Do you mind to add a comment explaining why we do this here (copy your PR description for example) ? For future reference... Thanks.

Reply:
Sorry about that. :wink: Comments added as requested.